### ISSUE TITLE
Inherits container bug

### ISSUE THREAD

**Issue by pix-art** [sonata-project/SonataPageBundle#745]
Hi all,

On Sonata 3.x there was a feature inherts_containers when defining templates. When implementing [2aa1cb3002e58ffe9d0493be5ca30003ec62a75b](https://github.com/sonata-project/SonataPageBundle/commit/2aa1cb3002e58ffe9d0493be5ca30003ec62a75b) commit the core code for this feature was removed. But the rest wasn't cleaned from documentation? So was this intentional? And if so why do the docs still show this? Or was this just a mistake?

The following code was removed:

```
if (isset($template['inherits_containers'])) {
    $inherits = $template['inherits_containers'];
    if (!isset($templates[$inherits])) {
        throw new InvalidConfigurationException(sprintf('Template "%s" cannot inherits containers from undefined template "%s"', $id, $inherits));
    }
    $template['containers'] = array_merge($templates[$inherits]['containers'], $template['containers']);
}
```

**Comment by greg0ire** [comment]
ping @rande

**Comment by jordisala1991** [comment]
Seems fixed in #747 . Closing
